domains/measurements/Measurements.API/Tests/Measurements/gRPC/V1/Services/MeasurementsServiceTests.GetMeasurements.verified.txt

```diff
@@ -0,0 +1,10 @@
+﻿{
+  Measurements: [
+    {
+      Gsrn: 1234567890123456,
+      DateFrom: 1609542000,
+      DateTo: 1609545600,
+      Quantity: 880
```

Comment:
Wont we need some more data from measurements when using the gRPC endpint?

Reply:
Not sure what you mean?

Reply:
Added specific assert for this